scripts/apitypings/main.go

```diff
@@ -28,13 +28,13 @@ import (
 
 var (
 	// baseDirs are the directories to introspect for types to generate.
-	baseDirs = [...]string{"./codersdk", "./coderd/healthcheck"}
+	baseDirs = [...]string{"./codersdk"}
 	// externalTypes are types that are not in the baseDirs, but we want to
 	// support. These are usually types that are used in the baseDirs.
 	// Do not include things like "Database", as that would break the idea
 	// of splitting db and api types.
 	// Only include dirs that are client facing packages.
-	externalTypeDirs = [...]string{"./cli/clibase", "./coderd/healthcheck/health", "./coderd/healthcheck/derphealth"}
+	externalTypeDirs = [...]string{"./cli/clibase", "./coderd/healthcheck/health"}
```

Comment:
review: no longer need to generate `coderd/healthcheck` and `coderd/healthcheck/derphealth`.